waterdata/views.py

```diff
@@ -206,6 +208,67 @@ def hydrological_unit_locations(huc_cd):
     return hydrological_unit(huc_cd, show_locations=True)
 
 
+@app.route('/states/', defaults={'state_cd': None, 'county_cd': None}, methods=['GET'])
+@app.route('/states/<state_cd>/', defaults={'county_cd': None}, methods=['GET'])
+@app.route('/states/<state_cd>/counties/<county_cd>/', methods=['GET'])
+@defined_when(app.config['STATE_COUNTY_PAGES_ENABLED'], return_404)
+def states_counties(state_cd, county_cd, show_locations=False):
+    """
+    State unit view
+
+    :param state_cd: ID for this political unit - 'state'
+    :param county_cd: ID for this political unit - 'county'
+    """
+
+    # Get the data associated with this county
+    if state_cd and county_cd:
+        state_county_cd = state_cd + county_cd
+        unit_data = app.config['COUNTRY_STATE_COUNTY_LOOKUP']['US']['state_cd'].get(state_cd, None)['county_cd'].get(county_cd, None)
+
+    # Get the data corresponding to this state
+    if state_cd and not county_cd:
+        unit_data = app.config['COUNTRY_STATE_COUNTY_LOOKUP']['US']['state_cd'].get(state_cd, None)
+
+    # If no state and or state and county code is available, display list of states.
+    if not state_cd and not county_cd:
+        unit_data = {
+            'name': 'US',
```

Comment:
Consider assigning the name to be 'United States' rather than having an if statement in the template.